TaskModules/powershell/TaskModuleSqlUtility/SqlPackageOnTargetMachines.ps1

```diff
@@ -422,7 +422,7 @@ function Get-SqlPackageCmdArgs
         {
             if($sqlServerCredentials)
             {
-                $sqlUsername = $sqlServerCredentials.GetNetworkCredential().username
+                $sqlUsername = $sqlServerCredentials.UserName
```

Comment:
Curious if GetNetworkCredentials().username not working, how GetNetworkCredentials().password working? If it is working why to change?